src/components/GraphiQL.js

```diff
@@ -694,6 +710,42 @@ export class GraphiQL extends React.Component {
     editor.setValue(print(parse(editor.getValue())));
   };
 
+  handleDefragQuery = () => {
+    const editor = this.getQueryEditor();
+    const ast = parse(editor.getValue());
```

Comment:
What if `editor.getValue()` returns empty/null? We need to guard against that

Reply:
Yes, you're right. And I just realized that we have a bug with this on the prettify feature